templates/js-crawlee-cheerio/.actor/input_schema.json

```
@@ -10,9 +10,15 @@
             "editor": "requestListSources",
             "prefill": [
                 {
-                    "url": "https://apify.com"
+                    "url": "https://crawlee.com"
```

I think this is not our domain 😄 
```suggestion
                    "url": "https://crawlee.dev"
```